src/test/java/com/exasol/adapter/dialects/exasol/AbstractRemoteExasolVirtualSchemaConnectionIT.java

```diff
@@ -0,0 +1,59 @@
+package com.exasol.adapter.dialects.exasol;
+
+import static com.exasol.matcher.ResultSetStructureMatcher.table;
+
+import org.junit.jupiter.api.Test;
+
+import com.exasol.dbbuilder.dialects.Table;
+
+/**
+ * This class contains the common parts of the variants of the Exasol Virtual Schema that connect to a remote Exasol
+ * instance or cluster.
+ */
+abstract class AbstractRemoteExasolVirtualSchemaConnectionIT extends AbstractExasolSqlDialectIT {
```

Comment:
Nitpick: As we have the `abstract` class modifier, do we also need it in the name? Can't we just name the class `RemoteExasolVirtualSchemaConnectionIT` ?